nav_msgs/Path path
---
bool succeded
string error_msg
